providers/factories/oracle/marketmap.go

```diff
@@ -33,49 +34,21 @@ func DefaultMarketMapProvider(
 		Transport: &http.Transport{MaxConnsPerHost: cfg.API.MaxQueries},
 		Timeout:   cfg.API.Timeout,
 	}
-	requestHandler, err := apihandlers.NewRequestHandlerImpl(client)
-	if err != nil {
-		return nil, err
-	}
+	var apiDataHandler types.MarketMapAPIDataHandler
+	var requestHandler apihandlers.RequestHandler
+	var ids []types.Chain
 
-	queryHandler, err := types.NewMarketMapAPIQueryHandler(
-		logger,
-		cfg.API,
-		requestHandler,
-		apiDataHandler,
-		apiMetrics,
-	)
-	if err != nil {
-		return nil, err
+	switch cfg.Name {
+	case dydx.Name:
+		apiDataHandler, err = dydx.NewAPIHandler(cfg.API)
+		ids = []types.Chain{{ChainID: dydx.ChainID}}
```

Comment:
I'm not sure how this is used right now, but it feels like we don't really need the chain-id as long as we point the mm provider at the correct node?

Reply:
Yea this is currently hard coded based on the desired chain but it doesnt really matter since all you have to do is point it to the correct URL. The providers were purposfully designed to assume you have inputs when constructing URLs, but in the future we can probably make it allow static or no entries.